app/modules/services/SettingsService/Tests/DefaultSettingsServiceTests.swift

```diff
@@ -37,9 +37,8 @@ struct DefaultSettingsServiceTests {
   @MainActor
   func test_apiKeyKeychainMapping() async throws {
     // Setup
-    let fileManager = MockFileManager()
```

Comment:
For this test case and several ones below, the convenience initializer already create mock objects by default so just use them as much as possible. This makes the tests more concise.